.storybook/stories/datagrid/datagrid-detail.stories.ts

```diff
@@ -154,3 +156,11 @@ const DetailTemplate: StoryFn = args => {
 export const Detail: StoryObj = {
   render: DetailTemplate,
 };
+
+export const OpenDetail: StoryObj = {
+  render: DetailTemplate,
```

Comment:
```suggestion
  render: DetailTemplate,
  play: removeFocusOutline,
```

I just realized the screenshots have a keyboard focus outline.